optuna_dashboard/llm/_prompt_templates.py

```diff
@@ -0,0 +1,110 @@
+from __future__ import annotations
+
+
+_TRIAL_DEFINITION_IN_TYPESCRIPT = """```typescript
+export type TrialState = "Running" | "Complete" | "Pruned" | "Fail" | "Waiting";
+export type TrialIntermediateValue = {
+    step: number;
+    value: number;
+};
+export type Attribute = {
+    key: string;
+    value: string;
+};
+export type Trial = {
+    trial_id: number;
+    study_id: number;
+    number: number;
+    state: TrialState;
+    values?: number[];  // ith-objective value is stored in values[i] (note: 0-indexed)
+    params: TrialParam[];
+    intermediate_values: TrialIntermediateValue[];
+    user_attrs: Attribute[];
+    datetime_start?: Date;
+    datetime_complete?: Date;
+    constraints: number[];
+};
+export type TrialParam = {
+    name: string;
+    param_internal_value: number;
+    param_external_value: string;
+    param_external_type: string;
+    distribution: Distribution;
+};
+```"""
+
+_TRIAL_FILTERING_PROMPT_TEMPLATE = """Please write a filtering function for the user query provided in Javascript.
```

Comment:
Nits: JavaScript is correct
```suggestion
_TRIAL_FILTERING_PROMPT_TEMPLATE = """Please write a filtering function for the user query provided in JavaScript.
```